src/pages/Home/Home.jsx

```diff
@@ -17,7 +16,6 @@ const Home = () => (
       <h2 className="title__nav-home">
         BOOK ROOMS, HOMES & APTS
       </h2>
-      <HotelFilter />
```

Comment:
Why did you delete filter from home page?

Reply:
is now back on home